main.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	clientpkg "awscleaner/pkg/aws"
+	"fmt"
+	_ "fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/aws/aws-sdk-go/service/sts"
+)
+
+const (
+
+	// the credentials for aws-account-shredder
+	accessID  = ""
+	secretKey = ""
+	accountid = "" // it is 12 digit account id , the sub level account , for future use and debugging purpose
+)
+
+func main() {
+
+	// creating a new cient with us-east-1 region by default
+	client, err := clientpkg.NewClient(accessID, secretKey, "", "us-east-1")
+	if err != nil {
+		fmt.Println("error is %v", err)
+	}
+	assumedRole, err := client.AssumeRole(&sts.AssumeRoleInput{RoleArn: aws.String("arn:aws:iam::173028519319:role/OrganizationAccountAccessRole"), RoleSessionName: aws.String("agautam")})
+	if err != nil {
+		fmt.Println("error is %v", err)
+	}
+	assumedAccessKey := *assumedRole.Credentials.AccessKeyId
+	assumedSecretKey := *assumedRole.Credentials.SecretAccessKey
+	assumedSessionToken := *assumedRole.Credentials.SessionToken
+
+	// for debugging purpose only.
+	//	fmt.Println("new access id : ", assumedAccessKey)
+	//	fmt.Println("new secret key is", assumedSecretKey)
+	//	fmt.Println("new session token\n\n", assumedSessionToken)
+
+	// looping through all the regions
+	regionRange := []string{"us-east-1", "us-east-2", "us-west-1", "us-west-2", "ca-central-1", "eu-central-1", "eu-west-1", "eu-west-2", "eu-west-3", "ap-northeast-1", "ap-northeast-2", "ap-south-1", "ap-southeast-1", "ap-southeast-2", "sa-east-1"}
+	for _, region := range regionRange {
+		fmt.Println("\n EC2 instances in region ", region)
+		client2, err := clientpkg.NewClient(assumedAccessKey, assumedSecretKey, assumedSessionToken, region)
```

Comment:
Minor nit, client2 to is not very descriptive. Maybe something like `assumedRoleClient`